includes/wikia/Extensions.php

```diff
@@ -94,6 +94,13 @@
 	F::app()->registerApiController( 'WikisApiController', "{$IP}/includes/wikia/api/WikisApiController.class.php" );
 }
 
+// During migration drop parser expiry on non-English wikis to 24 hours (PLATFORM-3765)
+if ( !wfHttpsAllowedForURL( $wgServer ) ||
+	( wfHttpsEnabledForURL( $wgServer ) && $wgLanguageCode !== 'en' )
+) {
+	$wgParserCacheExpireTime = 24 * 3600;
```

Comment:
Is there a specific reason it's in `Extensions.php`? Can it be set [earlier](https://github.com/Wikia/config#whats-the-order-in-which-the-configuration-is-loaded)? It's just weird as it's not an extension.

Reply:
Because it needs to be after WikiFactory is loaded, otherwise wgServer and wgLanguageCode may not be set yet.